documentation/manual/src/main/asciidoc/modules/mongodb.asciidoc

```diff
@@ -3473,6 +3473,55 @@ List<Poem> result = session.createNativeQuery( nativeQuery ).addEntity( Poem.cla
 ----
 ====
 
+[[ogm-mongodb-stored-proc-native]]
+
+==== Server-side JavaScript and stored procedures
+
+In MongoDB, it is possible to call server side javascripts as if it is a stored procedure.
```

Comment:
* In the main doc, you call it `server-side JavaScript`, let's be consistent about the case. Maybe use `server-side JavaScript scripts` here? It looks redundant but I think it's the way to go.
 * `as if they were stored procedures`